src/node.ts

```diff
@@ -385,46 +387,58 @@ export abstract class BaseNode<
    * @returns a mapping of name to the node visible in this node's scope.
    */
   public getLexicalScope(): Map<string, BindingDecl> {
-    return new Map(getLexicalScope(this as unknown as FunctionlessNode));
+    return new Map(
+      getLexicalScope(this as unknown as FunctionlessNode, "scope")
+    );
 
-    function getLexicalScope(node: FunctionlessNode | undefined): Binding[] {
+    function getLexicalScope(
+      node: FunctionlessNode | undefined,
+      kind: "scope" | "sibling"
```

Comment:
Document this parameter, what is a `sibling`?

Reply:
ahh, yeah, meant to take another look at this. Its kind of ugly.